wgengine/filter/filter_test.go

```diff
@@ -188,3 +256,28 @@ func rawpacket(proto packet.IPProto, len uint16) []byte {
 
 	return hdr
 }
+
+func tcpudp(proto packet.IPProto, src, dst packet.IP, sport, dport uint16) []byte {
```

Comment:
this function name+signature isn't obvious. Add a comment about what it's supposed to do?